### PR TITLE
Add Summary119120 + all-contributor additions

### PR DESCRIPTION
**Pull request by Davidutro**
No breaking changes

### CODE REVIEW

File: content/en/contribute/index.mdx [hunk 1]
```diff
@@ -190,3 +190,6 @@ We welcome ideas for improving our community resources. Submit a <Link to="/fund
 
 </Tout>
 
+## Our Contributor Community
+
+<Authors all/>
```
Comment (twblack88) [inline content/en/contribute/index.mdx:195]: There is some styling coming that should remove the block-ey feeling to all authors being displayed.

One thing that would be helpful is considering how many authors should display and then rotating them on some automatic frequency.

File: content/en/index.mdx [hunk 2]
```diff
@@ -112,3 +112,4 @@ Help us grow and make the MakerDAO community even better. Reach out in our chat
 
 </InfoBlock>
 
+<Authors all/>
```
Comment (twblack88) [inline content/en/index.mdx:115]: I would leave all authors on the contribute page, this should def be a community vote.

**Review by twblack88 — COMMENTED** [review]
Al contirbutors is getting a face lift so if you mind just taking those changes off @Davidutro then we can merge the .json and governance ones for sure. 

Also, I think it was suggested to vote on displaying folks via the contributor page and index page or just contributor page. That needs some clarity.

File: content/en/index.mdx (hunk 2 above)
Comment (Davidutro) [reply content/en/index.mdx:115]: https://chat.makerdao.com/channel/community-development?msg=FgdGPnSf2ipKdpP9y

There was a vote, and it resolved to be the bottom of the front page, that's why I had placed it there for the PR 😅

**Review by twblack88 — APPROVED** [review]

**Pull request merged**